starter-code/index.html

```diff
@@ -6,8 +6,317 @@
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>CSS Lab | Responsive Cloning | Product Hunt</title>
   <link rel="stylesheet" href="styles.min.css">
+  <link href="https://fonts.googleapis.com/css?family=Nunito" rel="stylesheet">
 </head>
 <body>
-  
+  <header class="header">
+    <div class="search-container">
+      <div class="search">
+       <img src="images/logo.png" alt="logo"><img src="images/search.png" class="img-lense" alt="search" /><input type="text" class="input-discover" placeholder="Discover your next favorite thing..." id="search">
+      Login
+      </div>
+    </div>
+    <nav class="navigation">
+      <div class="navigation-container">
```

Comment:
This navigation-container is totally needed ?
Try move his CSS rules to the .navigation so see what happens